tests/shap/test_log.py

```diff
@@ -106,6 +108,24 @@ def test_sklearn_log_explainer_pyfunc():
         np.testing.assert_allclose(shap_values_original.values, shap_values_new, rtol=100, atol=100)
 
 
+def test_log_explanation_doesnt_create_autologged_run():
+    mlflow.sklearn.autolog(disable=False, exclusive=False)
+    dataset = sklearn.datasets.load_boston()
+    X = pd.DataFrame(dataset.data[:50, :8], columns=dataset.feature_names[:8])
+    y = dataset.target[:50]
+    model = sklearn.linear_model.LinearRegression()
+    model.fit(X, y)
+
+    with mlflow.start_run() as run:
+        mlflow.shap.log_explanation(model.predict, X)
+
+    run_data = MlflowClient().get_run(run.info.run_id).data
+    metrics, params, tags = run_data.metrics, run_data.params, run_data.tags
+    assert not metrics
```

Comment:
can we also assert that the `mlflow.autologging` is absent?

Reply:
I think you mean the `mlflow.autologging` tag? sounds good